src/kbmod/search/image_stack.cpp

```diff
@@ -92,11 +92,6 @@ void ImageStack::sort_by_time() {
               [](const LayeredImage& a, const LayeredImage& b) { return a.get_obstime() < b.get_obstime(); });
 }
 
-void ImageStack::convolve_psf() {
-    if (data_on_gpu) throw std::runtime_error("Cannot modify images while on GPU");
-    for (auto& i : images) i.convolve_psf();
-}
-
```

Comment:
just out of curiousity, did you benchmark the pytorch cpu convolve vs our previous C++ impl?

Reply:
Yes. Here is what I got from my timings:
C++ GPU: 7.3 ms
C++ CPU: 1.55 s
PyTorch GPU: 5.8 ms
PyTorch CPU: 189 ms
SciPy's convolve2d: 1.43 s

Reply:
The PyTorch CPU is a lot slower than the PyTorch GPU, but MUCH faster than the C++ CPU.